pkgs/development/python-modules/zigpy-znp/default.nix

```diff
@@ -3,50 +3,53 @@
 , asynctest
 , buildPythonPackage
 , coloredlogs
-, coveralls
 , fetchFromGitHub
+, jsonschema
 , pyserial
 , pyserial-asyncio
 , pytest-asyncio
 , pytest-mock
 , pytest-timeout
-, pytestcov
 , pytestCheckHook
+, pythonOlder
 , voluptuous
-, zigpy }:
+, zigpy
+}:
 
 buildPythonPackage rec {
   pname = "zigpy-znp";
-  version = "0.4.0";
+  version = "0.5.1";
 
   src = fetchFromGitHub {
     owner = "zha-ng";
     repo = "zigpy-znp";
     rev = "v${version}";
-    sha256 = "1g5jssdnibhb4i4k1js9iy9w40cipf1gdnyp847x0bv6wblzx8rl";
+    sha256 = "sha256-lkaJ9xO25sxnhRLrfxovCNu1rAsk2qoskXNnJwdATZQ=";
   };
 
   propagatedBuildInputs = [
     async-timeout
     coloredlogs
+    jsonschema
     pyserial
     pyserial-asyncio
     voluptuous
     zigpy
   ];
 
   checkInputs = [
-    asynctest
-    coveralls
     pytest-asyncio
     pytest-mock
     pytest-timeout
-    pytestcov
     pytestCheckHook
+  ]  ++ lib.optionals (pythonOlder "3.8") [
+    asynctest
   ];
 
+  pythonImportsCheck = [ "zigpy_znp" ];
+
   meta = with lib; {
-    description = "A library for zigpy which communicates with TI ZNP radios";
+    description = "Python library for zigpy which communicates with TI ZNP radios";
     homepage = "https://github.com/zha-ng/zigpy-znp";
```

Comment:
The GitHub redirect is still valid for now but the correct URL is https://github.com/zigpy/zigpy-znp (the organization changed from `zha-ng` to `zigpy`). It seems like this would also apply to line 24.

Reply:
Thanks, updated.